sdks/python/apache_beam/runners/direct/consumer_tracking_pipeline_visitor_test.py

```diff
@@ -45,6 +45,10 @@ class ConsumerTrackingPipelineVisitorTest(unittest.TestCase):
   def setUp(self):
     self.pipeline = Pipeline(DirectRunner())
     self.visitor = ConsumerTrackingPipelineVisitor()
+    try:                    # Python 2
+      self.assertCountEqual = self.assertItemsEqual
```

Comment:
Would not this raise an AttributeError in any case?
python 2 does not have the first method, python 3 does not have the second one.

Reply:
Nevermind, on py2, this will define assertCountEqual.